### ISSUE TITLE
[Bug Report] [CSS] No padding left on 2nd tab label when tabs are nested. 

### ISSUE THREAD

**Issue by mathieutu** [ElemeFE/element#7987]
### Element UI version
2.0.2

### OS/Browsers version
Any

### Vue version
2.5.2

### Reproduction Link
[https://jsfiddle.net/6pn1m6xb/2/](https://jsfiddle.net/6pn1m6xb/2/)

### Steps to reproduce
See the reproduction link:
When tabs are nested like in my example, the 2nd tab label of any nested tabs set has a padding left = 0.

This is because of:
```css
/* theme-chalk.css */
el-tabs--top:not(.el-tabs--border-card):not(.el-tabs--card) .el-tabs__item:nth-child(2),
.el-tabs--bottom:not(.el-tabs--border-card):not(.el-tabs--card) .el-tabs__item:nth-child(2) {
  padding-left: 0px;
}
```
from:
https://github.com/ElemeFE/element/blob/7d7d5a87579440776dba9690dc71358c57036187/packages/theme-chalk/src/tabs.scss#L227-L231

### What is Expected?
Padding-left should have 20px, like other tabs label.

### What is actually happening?
Padding-left is setted at 0px.

**Comment by mathieutu** [comment]
Ok, nevermind, I've just seen this new PR: 
https://github.com/ElemeFE/element/pull/7941

I will close this issue when the update will be released.

Thank's @Leopoldthecoder!

**Comment by Leopoldthecoder** [comment]
@mathieutu I thought #7941 fixes this too but only to find that it didn't take care of tabs with `tab-position` set to left or right. A valid fix is https://github.com/ElemeFE/element/pull/8015/files#diff-b2f475dc8dd81d5147490b064cd44cd5 .

I'm sorry you'll have to wait for the next release.

**Comment by mathieutu** [comment]
No problem, thanks for the answer! 
Just to know, do you have any idea of when it will be released ?

**Comment by Leopoldthecoder** [comment]
Around next Wednesday

**Comment by mathieutu** [comment]
👌 
Have some other issues to open (especially about the size in form) but don't have time for now to formalize that. Maybe I will try to do that before the next release...

**Comment by Leopoldthecoder** [comment]
Fixed in 2.0.4
